pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -117,175 +121,151 @@ func labelMachineSetNodes(client runtimeclient.Client, ms *mapiv1beta1.MachineSe
 	})
 }
 
+// Build default CA resource to allow fast scaling up and down
+func clusterAutoscalerResource() *caov1alpha1.ClusterAutoscaler {
```

Comment:
can we move this into utils?

Reply:
I prefer to keep it in the same file as it is autoscaling specific. Unless it is needed in a different `e2e/PACKAGE` suite. The less we expose/export the better.

Reply:
I think there's a lot to be said in having it scoped locally if it is only used locally, so +1 from me.

Reply:
I was suggesting pkg/e2e/autoscaler/utils.go so no details are exposed on the ginkgo file, so it's smaller and just reads user stories. Not a blocker for me though